types/src/account_address.rs

```diff
@@ -65,6 +65,26 @@ impl AccountAddress {
         keccak.finalize(&mut hash);
         AccountAddress::new(hash)
     }
+
+    pub fn from_hex_literal(literal: &str) -> Result<Self> {
+        let mut hex_string = String::from(&literal[2..]);
+        if hex_string.len() % 2 != 0 {
+            hex_string.insert(0, '0');
+        }
+
+        let mut result = hex::decode(hex_string.as_str()).unwrap();
```

Comment:
Propagate the error here rather than unwrap?

Reply:
Done